deploy/template.yaml

```diff
@@ -635,6 +647,56 @@ Resources:
       FilterPattern: ""
       LogGroupName: !Ref APIGWAccessLogsGroup
 
+  AddressFE5XXErrorCriticalAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: DeployAlarms
+    Properties:
+      AlarmName: !Sub "${AWS::StackName}-${Environment}-AddressFE5XXErrorCriticalAlarm"
+      AlarmDescription: !Sub "Threshold exceeds 80% with a minimum of 20 invocations and a minimum of 5 errors in 2 out of the last 5 evaluation periods. ${SupportManualURL}"
+      ActionsEnabled: true
+      AlarmActions:
+        - !ImportValue core-infrastructure-AlarmTopic
+        - !ImportValue platform-alarm-critical-alert-topic
+      OKActions:
+        - !ImportValue core-infrastructure-AlarmTopic
+        - !ImportValue platform-alarm-critical-alert-topic
+      EvaluationPeriods: 5
+      DatapointsToAlarm: 2
+      Threshold: 80
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      TreatMissingData: notBreaching
+      Metrics:
+        - Id: errorThreshold
+          Label: errorThreshold
+          ReturnData: true
+          Expression: IF(invocations < 20 || error < 5, 0, errorPercentage)
```

Comment:
This can just be `IF(invocations<20,0,errorPercentage)`
We only want to default to 0 if innovations is less than our set amount, no need to default under an error percentage